### ISSUE TITLE
[Other - Accessibility] Hint text likely to be missed by screen reader users. (00.00.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#72161]
#### VFS team name
Veteran Facing Forms

#### VFS product name
20-10206

#### VFS feature name
FOIA / PA request

#### Point of Contact/Reviewers
Brian DeConinck - @briandeconinck - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
No VA.gov Experience Standard for the issue found.

### Issue Details
On [/record-selections](https://staging.va.gov/records/request-personal-records-form-20-10206/record-selections), the hint text for the checkbox group isn't likely to be heard by screen reader users in forms mode or navigating the form just by tabbing. This instance isn't critical, since it's not telling users anything that's particularly actionable. But it is setting expectations (depending on what you choose, this might take longer) which could influence decision-making around what a user requests. Should be available to everyone.

### VA.gov Experience Standard
[Category Number 00, Issue Number 00](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Platform Recommendation
Jeana and Tze already determined this to be a design system component bug and [filed an issue](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2342) there. No recommended action until the DST provides an update. Logging here just for completeness.

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by tbaker1026** [comment]
Leave open until design system fixes it.  Once fixed test in our form and close it.

**Comment by jeana-adhoc** [comment]
The Design System team fixed the bug with the [hint text in this ticket](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2342)- this can be closed.
